src/app/clusters/time-synchronization-server/time-synchronization-server.h

```diff
@@ -123,6 +123,8 @@ class TimeSynchronizationServer : public FabricTable::Delegate
     // ReadClient::Callback functions
     void OnAttributeData(const ConcreteDataAttributePath & aPath, TLV::TLVReader * apData, const StatusIB & aStatus) override;
     void OnDone(ReadClient * apReadClient) override;
+
+    CHIP_ERROR AttemptToGetTimeFromTrustedNode();
```

Comment:
why is this public now?

Reply:
It is so that apps can reuse this for periodic/random querying of the TrustedTimeSource. We use it to query time once a day. Just didn't want to write code which basically does the same thing.